pkg/kubectl/cmd/util/factory.go

```diff
@@ -682,11 +667,16 @@ func (f *factory) Scaler(mapping *meta.RESTMapping) (kubectl.Scaler, error) {
 
 func (f *factory) Reaper(mapping *meta.RESTMapping) (kubectl.Reaper, error) {
 	mappingVersion := mapping.GroupVersionKind.GroupVersion()
-	clientset, err := f.clients.ClientSetForVersion(&mappingVersion)
-	if err != nil {
-		return nil, err
+	clientset, clientsetErr := f.clients.ClientSetForVersion(&mappingVersion)
+	reaper, reaperErr := kubectl.ReaperFor(mapping.GroupVersionKind.GroupKind(), clientset)
```

Comment:
it's intentional to call `ReaperFor` despite of a clientsetErr? If yes, please add a comment why.